src/build/cargo.rs

```diff
@@ -294,37 +293,25 @@ impl Executor for RlsExecutor {
             }
         }
 
-        let rustc_exe = env::var("RUSTC").unwrap_or(env::args().next().unwrap());
-        let mut cmd = Command::new(&rustc_exe);
+        // Cargo should handle `$RUSTC` wrapper override already
+        let mut cmd = cargo_cmd.clone();
```

Comment:
So on top of that, we'd still need some API for `ProcessBuilder` to change the program, right? And then we'd need to substitute with previous `rustc_exe`?